apps/pie-docs/test/helpers/playwright-helper.js

```diff
@@ -0,0 +1,22 @@
+const COOKIE_NAMES = require('../../../../constants/cookies');
+
+exports.disableCookieBanner = async (page, context) => {
+    const url = new URL(process.env.BASE_URL);
+    const hostName = url.hostname;
+    const pathName = '/';
+
+    await context.addCookies([
+        {
+            name: COOKIE_NAMES.JE_COOKIE_CONSENT,
+            value: 'full',
+            domain: hostName,
+            path: pathName,
+        }, {
+            name: COOKIE_NAMES.JE_BANNER_COOKIE,
+            value: '130315',
+            domain: hostName,
+            path: pathName,
+        }
+    ]);
+    await page.reload();
```

Comment:
I'm in two minds about this reload. I understand this reload means the cookie banner won't appear, but we're then navigating to a new page in each nav test which adds a lot more time to the tests on some machines. 

In the nav tests, going to the route to test after this call would have the same effect I think?

Reply:
We could always default to refreshing but pass in an optional bool to opt out. That could speed up the nav tests and keep it simple for other tests?